PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalVsPhos.cxx

```diff
@@ -1041,6 +1169,145 @@ void AliAnalysisTaskEmcalVsPhos::FillClusterHistograms()
 }
 
 /*
+ * This function fills the histograms for the jet performance study.
+ */
+void AliAnalysisTaskEmcalVsPhos::FillJetPerformanceHistograms()
+{
+  TString histname;
+  
+  // If MC, get the MC event
+  const AliMCEvent* mcevent;
```

Comment:
same as above (even if MCEvent will return valid pointer for sure it is better to be safe)

Reply:
Hi Adrian,
Thanks for the comment, I will make a fix momentarily.